src/stemmablenote.ts

```diff
@@ -243,7 +245,7 @@ export abstract class StemmableNote extends Note {
   }
 
   hasFlag(): boolean {
-    return Tables.getGlyphProps(this.duration).flag && !this.beam;
+    return this.renderFlag && Tables.getGlyphProps(this.duration).flag && !this.beam;
```

Comment:
`renderFlag` muddies the logic here -- I prefer to have something more specific (like the `!this.beam` in this line, so it's clear that if there's a beam, there's no flag.)

Reply:
`renderFlag` is used to avoid the rendering of the flag in shared note heads. Would you prefer something like `sharedNoteHead` and use it directly in `draw`?

Reply:
two notes with different pitches from different voices that get to share the stem and flag as a chord. This is coded in the VexFlow patch, I believe.

Reply:
I have decided to remove this feature.